ote/src/cljs/ote/views/transit_visualization.cljs

```diff
@@ -297,7 +301,24 @@
       (= (:gtfs/route-hash-id-type route-hash-id-type) "short-long-headsign")
       (= (:gtfs/route-hash-id-type route-hash-id-type) "long-headsign")))
 
-(defn route-changes [e! route-changes no-change-routes selected-route route-hash-id-type]
+(defn- route-change-summary
+  "Route list has first change row of that route change. To be able to show summary of changes in all route rows we
+  need to merge-with them together."
+  [single-change all-changes]
+  (let [selected-change-keys #{:removed-trips :trip-stop-sequence-changes-lower
+                               :trip-stop-sequence-changes-upper :route-hash-id
+                               :trip-stop-time-changes-lower :trip-stop-time-changes-upper :change-type :added-trips}
+        all-changes (map #(select-keys % selected-change-keys) all-changes)
+        single-change (select-keys single-change selected-change-keys)
+        all-route-changes (filter
+                            (fn [c]
+                              (let [x (select-keys c selected-change-keys)]
```

Comment:
Is the `let` unnecessary if function only examines two explicitly defined keys in the following condition?